app/containers/RoomItem/Actions.tsx

```diff
@@ -141,6 +141,7 @@ export const RightActions = React.memo(({ transX, favorite, width, toggleFav, on
 					styles.actionRightButtonContainer,
 					{
 						width: width * 2,
+						height: '100%',
```

Comment:
Hide and fav actions should be almost the same.
Have you tried adding `viewHeight` (and removing `isCondensed && { height: rowHeightCondensed },`) instead of `height 100%`?